src/controllers/appointment.ts

```diff
@@ -0,0 +1,23 @@
+import { NextFunction, Request, Response } from 'express';
+import getAppointmentsPerDateService from '../services/appointment';
+import { templateErrors } from '../helpers';
+
+const getAppointments = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  try {
+    const date = req.query.date as string;
+    const { therapistId } = req.params;
+
+    if (Number.isNaN(+therapistId)) throw templateErrors.BAD_REQUEST('not found');

```

Comment:
the error message should be clearer, indicating that the therapist ID is invalid rather than "not found"
```suggestion
    if (Number.isNaN(+therapistId)) throw templateErrors.BAD_REQUEST('Id must be a number');

```